system-apps/web-browser-app/webClient/src/app/browser/address-bar/address-bar.component.ts

```diff
@@ -0,0 +1,116 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+
+  SPDX-License-Identifier: EPL-2.0
+
+  Copyright Contributors to the Zowe Project.
+*/
+
+import { Component, OnInit, OnDestroy, HostBinding } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Subscription } from 'rxjs';
+import { throttleTime } from 'rxjs/operators';
+import { NavigationService, ProxyService } from '../services';
+import { SettingsService } from '../services/settings.service';
+
+@Component({
+  selector: 'app-address-bar',
+  templateUrl: './address-bar.component.html',
+  styleUrls: ['./address-bar.component.scss']
+})
+export class AddressBarComponent implements OnInit, OnDestroy {
+  urlControl: FormControl;
```

Comment:
The member is public because it is used in a corresponding HTML template. It matters for AoT compiler that runs when we run `npm run i18n`. See https://angular.io/guide/aot-compiler#public-symbols for details.